modules/kargoBidAdapter.md

```diff
@@ -0,0 +1,23 @@
+# Overview
+
+**Module Name**: Kargo Bidder Adapter
+**Module Type**: Bidder Adapter
+**Maintainer**: kraken@kargo.com
+
+# Description
```

Comment:
You might want to mention something about how this only works on mobile.

I tried this and thought I wasn't getting a bid back... but I put my browser in mobile mode and then it worked. Didn't realize that was a dependency. Others may run into the same problem.

Reply:
I updated this as well but its below the line commented on so Github didnt dismiss it